styles.css

```diff
@@ -975,34 +1100,39 @@ ol a {
     position: absolute;
   }
 
-  .header img {
-    display: none;
-  }
-
   .header__nav {
-    width: 99%;
+    background-color: rgba(255, 255, 255, 0);
+    width: 100%;
+    height: min-content;
     visibility: visible;
     display: flex;
     flex-direction: row;
     justify-content: center;
+    position: fixed;
   }
```

Comment:
- [ ] I suggest that you consider that the height of the mobile menu covers 100% height of the screen. This is to ensure that we try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=XXaeGqDDiMDPDoAc-0) and have a mobile menu with good user Interface ☺️.